src/modules/main/navbar/Navbar.tsx

```diff
@@ -35,28 +40,30 @@ const Navbar = ({ basePath }: Navbar.Props) => {
 
   return (
     <nav className={csx.navbar}>
-      <div className={csx.breadcrumbs}>
-        {breadcrumbs.map((breadcrumb, idx) => (
-          <React.Fragment key={idx}>
-            <div className={csx.breadcrumb}>
-              <Link to={getLinkPath(idx, breadcrumbs, basePath)}>{breadcrumb}</Link>
-            </div>
-            <div className={csx.breadcrumbDivider}>{'>'}</div>
-          </React.Fragment>
-        ))}
-      </div>
+      {width > TABLET_WIDTH && (
+        <div className={csx.breadcrumbs}>
+          {breadcrumbs.map((breadcrumb, idx) => (
+            <React.Fragment key={idx}>
+              <div className={csx.breadcrumb}>
+                <Link to={getLinkPath(idx, breadcrumbs, basePath)}>{breadcrumb}</Link>
+              </div>
+              <div className={csx.breadcrumbDivider}>{'>'}</div>
+            </React.Fragment>
+          ))}
+        </div>
+      )}
```

Comment:
You should return empty <div/> or "" by inline condition I think. Now you are returning null when the condition equal false.

Reply:
what's the point in returning empty string here if it's not string dependent condition